autofill/autofill-impl/src/main/java/com/duckduckgo/autofill/impl/ui/settings/AutofillSettingsActivity.kt

```diff
@@ -0,0 +1,220 @@
+                        binding.autofillAvailable.importPasswordsOption.isVisible = state.canImportFromGooglePasswords
+                    }
+                }
+        }
+
+        lifecycleScope.launch {
+            viewModel.commands.collectLatest {
+                when (it) {
+                    is NavigatePasswordList -> {
+                        globalActivityStarter.start(
+                            this@AutofillSettingsActivity,
+                            AutofillPasswordsManagementScreen(source = AutofillSettings),
+                        )
+                    }
+
+                    ImportPasswordsFromGoogle -> {
+                        launchImportPasswordsScreen()
+                    }
+
+                    NavigateToHowToSyncWithDesktop -> {
+                        launchImportPasswordsFromDesktopSyncScreen()
+                    }
+
+                    AskToConfirmResetExcludedSites -> {
+                        askToConfirmResetExcludedSites()
+                    }
+                }
+            }
+        }
+    }
+
+    private fun configureViewListeners() {
+        binding.autofillAvailable.autofillEnabledToggle.setOnCheckedChangeListener(globalAutofillToggleListener)
+        binding.autofillAvailable.passwordsListItem.setOnClickListener {
+            viewModel.onPasswordListClicked()
+        }
+        binding.autofillAvailable.importPasswordsOption.setOnClickListener {
+            viewModel.onImportPasswordsClicked()
+        }
+        binding.autofillAvailable.syncDesktopPasswordsOption.setOnClickListener {
+            viewModel.onImportFromDesktopWithSyncClicked()
+        }
+        binding.autofillAvailable.excludedSitesOption.setOnClickListener {
+            viewModel.onResetExcludedSitesClicked()
+        }
+    }
+
+    private fun configureInfoText() {
+        binding.autofillAvailable.autofillInfoText.addClickableLink(
+            annotation = "learn_more_link",
+            textSequence = binding.root.context.prependIconToText(
+                R.string.autofillSettingsAutofillSubtitle,
+                R.drawable.ic_lock_solid_12,
+            ).toSpanned(),
+            onClick = {
+                launchHelpPage()
+            },
+        )
+    }
+
+    private fun launchHelpPage() {
+        globalActivityStarter.start(
+            this,
+            WebViewActivityWithParams(
+                url = LEARN_MORE_LINK,
+                screenTitle = getString(R.string.credentialManagementAutofillHelpPageTitle),
+            ),
+        )
+    }
+
+    private fun launchImportPasswordsScreen() {
+        val dialog = ImportFromGooglePasswordsDialog.instance()
+        dialog.show(supportFragmentManager, IMPORT_FROM_GPM_DIALOG_TAG)
+    }
+
+    private fun launchImportPasswordsFromDesktopSyncScreen() {
+        globalActivityStarter.start(this, ImportPasswordActivityParams)
+    }
+
+    private fun askToConfirmResetExcludedSites() {
+        this?.let {
```

Comment:
nit; `this` is already non-null so don't need `?.` 

also unsure why you're using the `.let{ }` here at all, tbh. slightly more readable to ditch it IMO.